README.md

```diff
@@ -133,3 +133,206 @@ SimpleCov.start
 ```
 
 You can see your test coverage when you run your tests. If you want this in a graphical form, uncomment the `HTMLFormatter` line and see what happens!
+
```

Comment:
Extremely thorough run through provided on the Readme, provides clear outline of technologies utilised and steps taken to drive design.